lib/zclient.h

```diff
@@ -356,6 +360,7 @@ struct zclient {
 #define ZAPI_MESSAGE_SRCPFX   0x20
 /* Backup nexthops are present */
 #define ZAPI_MESSAGE_BACKUP_NEXTHOPS 0x40
+#define ZAPI_MESSAGE_NHG 0x80
```

Comment:
so this is using up the last bit in the 'message' flags octet - should at least be worth a comment?

Reply:
What should I comment exactly? api.message is a uint32_t so it has some room left still for more types.